src/Goerli-DssSpell.sol

```diff
@@ -50,11 +52,28 @@ contract DssSpellAction is DssAction {
     address constant MCD_CLIP_WBTC_B        = 0x4F51B15f8B86822d2Eca8a74BB4bA1e3c64F733F;
     address constant MCD_CLIP_CALC_WBTC_B   = 0x1b5a9aDaf15CAE0e3d0349be18b77180C1a0deCc;
 
+    // --- Offboarding: Current Liquidation Ratio ---
+    uint256 constant CURRENT_AAVE_MAT       =  165 * RAY / 100;
+    uint256 constant CURRENT_BAL_MAT        =  165 * RAY / 100;
+    uint256 constant CURRENT_COMP_MAT       =  165 * RAY / 100;
+
+    // --- Offboarding: Target Liquidation Ratio ---
+    uint256 constant TARGET_AAVE_MAT        = 1000 * RAY / 100;
+    uint256 constant TARGET_BAL_MAT         = 1600 * RAY / 100;
+    uint256 constant TARGET_COMP_MAT        =  500 * RAY / 100;
+
     constructor() public {
         WBTC = DssExecLib.getChangelogAddress("WBTC");
         PIP_WBTC = DssExecLib.getChangelogAddress("PIP_WBTC");
```

Comment:
Let's move this two inside the `actions()` as a regular variable.

Reply:
Done in e023fa6.